### DISCUSSION TITLE
Enhancing code security with the power of GitHub Copilot [SEC1068G]

### DISCUSSION THREAD

**Discussion by jkcso** [community/community#74655] [Copilot Conversations]
## Enhancing code security with the power of GitHub Copilot [SEC1068G]

**Speaker:** @jkcso - Joseph Katsioloudes
**Social:** You can follow me on [Linkedin](https://www.linkedin.com/in/jkcso/), [Twitter](https://twitter.com/jkcso) and [GitHub](https://github.com/)

Artificial intelligence (AI) is already like a copilot in our daily lives, acting as a digital assistant and providing personalized experiences. Though despite progress in many other areas, AI has historically stopped short of improving software development practices. Enter GitHub Copilot: The software industry’s first AI solution that distills the collective knowledge of the world’s developers into an editor extension that suggests code in real-time. This makes the process of building great software easier and faster. But what about more secure? Attend our session to find out.

Thank you for joining my/our session/workshop! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do my best to answer. 

To watch the recording, please click [here](https://youtu.be/mjjBVZvZDxA?si=sXbrwVrFHCOMs8JB).👈

**Comment by jkcso** [comment]
Thank you SO MUCH for attending my session! Here's the [video recording](https://youtu.be/mjjBVZvZDxA?si=sXbrwVrFHCOMs8JB).

I'm at your disposal for your questions, comments, feedback, anything!

**Reply by keeplearning-rgb** [reply]
Hi @jkcso 
I was going through your Session recording and tried few things with copilot.

As mentioned in your video, we provide the context and let copilot generate code for us it will be "Bullet-proof"

I observed that it is generating a code which is vulnerable to SQLi.
Correct me if i am wrong but this is my observation.

Please provide your insights on this.
Thank you

**Reply by jkcso** [reply]
Show me! And in the meantime please notice that I highlighted several times over the session that you should follow best practices, such as security testing for all your code, including the code that has been suggested to you. In addition, as the session shows, we are doing our best for as secure suggestions as possible, following always the best security practices we recommend!
